src/workers/notifications/utils.ts

```diff
@@ -80,6 +80,7 @@ export const buildPostContext = async (
       post,
       source: await post.source,
       sharedPost,
+      initiatorId: post.authorId,
```

Comment:
Adding it here will affect some other workers/notifications, for example moderators might not get notifications from blocked users, can you check?